lib/stock.js

```diff
@@ -18,6 +18,8 @@ const fields = [
   { header: 'Tx List (Insight)', name: 'txListUrl', parse: String },
   { header: 'Tx List (API)', name: 'txListApi', parse: String },
   { header: 'Funding Tx', name: 'fundingTx', parse: String }
+  { header: 'Sold (ScottCoin)', name: 'soldScottcoinTotal', parse: Number }
```

Comment:
Nice try! Unfortunately a couple of issues -
* Those columns are identified by `N` and `P` but the range on line 5 only pulls in `A:M`
* Those fields use some horrible macro in the sheet to update once every never. Ideally we would pull it from the insight API directly (e.g. the URL specified in column `L`)